api/android/README.md

```diff
@@ -5,36 +5,36 @@
 We assume that you already have experienced Android application developments with Android Studio.
 
  * Host PC:
-   * OS: Ubuntu 16.04 x86_64 LTS
+   * OS: Ubuntu 16.04 / 18.04 x86_64 LTS
    * Android Studio: Ubuntu version
    * Android SDK: Min version 24 (Nougat)
    * Android NDK: Use default ndk-bundle in Android Studio
    * GStreamer: gstreamer-1.0-android-universal-1.16.2
 
 ## Build library
 
-#### Environment variables
+### Environment variables
 
 First of all, you need to set-up the development environment as following:
 
 ```bash
-$ export ANDROID_DEV_ROOT=$HOME/android               # Set your own path (The default path will be "$HOME/Android".)
+$ export ANDROID_DEV_ROOT=$HOME/android           # Set your own path (default location: $HOME/Android)
 $ mkdir -p $ANDROID_DEV_ROOT/tools/sdk
 $ mkdir -p $ANDROID_DEV_ROOT/tools/ndk
 $ mkdir -p $ANDROID_DEV_ROOT/gstreamer-1.0
 $ mkdir -p $ANDROID_DEV_ROOT/workspace
 $
 $ vi ~/.bashrc
-# Environment variables for developing a NNStreamer application
+# The environment variables to develop an Android application with NNStreamer
 #
 export JAVA_HOME=/opt/android-studio/jre            # JRE path in Android Studio
-export ANDROID_DEV_ROOT=$HOME/android               # Set your own path (The default path will be "$HOME/Android".)
-#
-# $ANDROID_DEV_ROOT/tools/sdk                    # Android SDK root directory (default location: $HOME/Android/Sdk)
-# $ANDROID_DEV_ROOT/tools/ndk                    # Android NDK root directory (default location: $HOME/Android/Sdk/ndk/<ndk-version>)
-# $ANDROID_DEV_ROOT/gstreamer-1.0                # GStreamer binaries
-# $ANDROID_DEV_ROOT/workspace/nnstreamer         # NNStreamer cloned git repository
-#
+export ANDROID_DEV_ROOT=$HOME/android               # Set your own path (default location: "$HOME/Android".)
```

Comment:
@again4you , If we try to use '**android**', NNStreamer users will be often confused due to similar word (e.g. **android** , **Android** [default]).  So, I want  you to use "**Android** [default]", or use  unique name such as "**myandroid**" or "**Android_nnstreamer**".

Reply:
ACK!